config/development.json

```diff
@@ -31,6 +31,8 @@
 		"ADMIN_PASSWORD": "dBildungscloud"
 	},
 	"INCOMING_REQUEST_TIMEOUT_API": 60000,
+	"INCOMING_REQUEST_TIMEOUT_COPY_API": 60000,
+	"FEATURE_COURSE_COPY_ENABLED": true,
```

Comment:
It exist a cleanup ticket for the copy changes. We must check the current state directly for the merge.

Reply:
This row shouldn't be in here anyway, should it? We needed it for testing, but we shouldn't merge line 35 in production.

Reply:
What was the decision here?

Reply:
`"FEATURE_COURSE_COPY_ENABLED": true` was removed from PR